plugin/src/main/java/org/jenkinsci/plugins/structs/describable/DescribableModel.java

```diff
@@ -427,8 +493,27 @@ private Object coerce(String context, Type type, Object o) throws Exception {
      * @param o a data-bound object
      * @return constructor and/or setter parameters
      * @throws UnsupportedOperationException if the class does not follow the expected structure
+     * @deprecated as of 1.2
+     *      Use {@link #uninstantiate2(Object)}
      */
     public Map<String,Object> uninstantiate(T o) throws UnsupportedOperationException {
+        return uninstantiate2(o).toMap();
```

Comment:
(`instantiate` may accept either the new or the old format. But `uninstantiate` must return the original format.)